Makefile

```diff
@@ -195,7 +195,7 @@ ifneq ($(HAS_WEBAPP),)
 	mkdir -p dist/$(PLUGIN_ID)/webapp
 	cp -r webapp/dist dist/$(PLUGIN_ID)/webapp/
 	## We remove the assets directory since files are redundant and loaded from webapp.
-	rm -fr widget/dist/files
+	rm -fr widget/dist/files/*.png
```

Comment:
Using the existing assets from webapp became problematic but going completely standalone means that webpack will include tons of stuff that's not really needed, some of which are 20MB pngs with the whole emoji set. I need to find a proper solution to this but for now just removing the heavy unused stuff to keep the build to a reasonable size.